src/collection-preferences/interfaces.ts

```diff
@@ -132,6 +134,8 @@ export interface CollectionPreferencesProps<CustomPreferenceType = any> extends
    *   - `editable` (boolean) - (Optional) Determines whether the user is able to toggle its visibility. This is `true` by default.
    *
    * You must set the current list of visible content `id`s in the `preferences.visibleContent` property.
+   *
+   * **Deprecated** in table, replaced by `contentDisplayPreference`.
```

Comment:
I'm not using the proper `@deprecated` tag here because the property is only deprecated in table.